db/migrate/20160915160023_add_allow_cloud_tenant_mapping_to_ext_management_systems.rb

```diff
@@ -0,0 +1,5 @@
+class AddAllowCloudTenantMappingToExtManagementSystems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :allow_cloud_tenant_mapping, :boolean
```

Comment:
@lpichler Perhaps a better name for this would be `tenant_mapping_enabled`. Not sure if we need `cloud` in the name, we may end up doing this for an infrastructure provider at some point.